frontend/src/pages/ExperimentList.test.tsx

```diff
@@ -179,10 +178,7 @@ describe('ExperimentList', () => {
   it('calls Apis to list experiments with namespace when available', async () => {
     await mountWithNExperiments(1, 1, { namespace: 'test-ns' });
     expect(listExperimentsSpy).toHaveBeenLastCalledWith(
-      '',
-      10,
-      'created_at desc',
-      '',
```

Comment:
nit: not sure if this is just a common UI practice, feels a bit unreadable to me. Does it make sense to have a wrapper function with a more meaningful name?

Reply:
I'm not sure what confused you. Do you mean this `expect(xxxSpy).toHaveBeenLastCalledWith(...some arguments)`?

This is fairly common, but previous implementation has been listing arguments (even though they are just default values).
I think if we change signature of listExperiment api method to let it accept an object of arguments, then the verifications here will include argument names, it will be a lot more readable.
e.g. proposed change will look like
```
expect(listExperimentsSpy).toHaveBeenLastCalledWith({
  page_token: xxx,
  page_size: xxx,
  sort_by: xxx,
  filter: xxx,
  resource_reference_key_type: xxx,
  resource_reference_key_id: xxx,
})
```

Above is a little more than what we need to do in this PR, so I will leave that to future. What do you think?

Reply:
Sorry, somehow my comment was placed right above the line that I wanted to comment on. 
I meant `...LIST_EXPERIMENT_DEFAULTS.slice(0, 4)` wasn't very clear to me what it's trying to get with the `slice` part. I was thinking maybe have a helper function to hide it away, something like:
```
function getTestExperimentWithoutNamespace() {
  return [...LIST_EXPERIMENT_DEFAULTS.slice(0, 4)]
}
```
or maybe even better as:
```
function getExperimentForTest(namespace='test-ns') {
  return [...LIST_EXPERIMENT_DEFAULTS.slice(0, 4), 'NAMESPACE', namespace]
}
```
Does it make sense? Just a nitpick. :)

Reply:
Thanks for the explanation!
I don't quite like introducing another function, to keep it simple, I will just add another constant. please take a look what I changed

Reply:
Yes, this looks good :)